src/courses/delta/README.md

```diff
@@ -0,0 +1,22 @@
+---
+order: 01
+next: 02.md
+title: Delta Process Developer Class
+author: Daniel Medina, George Dias
+---
+
+## 1.1 Class Overview
+
+The purpose of this class is to demonstrate how the MITRE SAF Delta tool can be used to facilitate changes to InSpec profile baselines. This process is often utilized to create a new baseline profile from older revisions of the same baseline.
+
+### 1.1.1 Class Objectives
+
+By the end of this class, you should be able to:
+
+- Identify the requirements for conducting the delta process
+- Understand the purpose and functionality of the SAF Delta process tools: `update_controls4delta` and `delta`
+- Utilize these tools to update InSpec profiles
+- Determine how and where to apply these tools in various scenarios (Use Cases)
+- Use the delta fuzzy matching system to map controls between two profiles
+- Become familiar with the delta process logging capabilities and the artifacts it generates
+- Format InSpec profile controls appropriately before and/or after executing delta commands
```

Comment:
I would add a What is Delta and Why are we talking about it idea, like the InSpec 101 from the developer class on page 1 of the beginner developer class or like the goal of the SAF on page 2 of the User class. We need to introduce the WHAT and WHY before we start talking about the HOW in page 2 here.